swap/src/cli/tracing.rs

```diff
@@ -23,18 +23,19 @@ pub fn init(debug: bool, json: bool, dir: impl AsRef<Path>, swap_id: Option<Uuid
         let file_logger = fmt::layer()
             .with_ansi(false)
             .with_target(false)
+            .json()
             .with_writer(appender);
 
         if json && debug {
```

Comment:
Testing for the `json` flag is now pointless here, isn't it?

Reply:
I don't think so. We just want the file appender to always use JSON. The default terminal printer should still adhere to whether the JSON flag was passed or not. I might be misunderstanding something here.

Reply:
We are always passing the same `file_logger` though so I think that can be extracted?

Reply:
Like this? Can this be simplified further?